careerbuilder/pencilblue
# [ACEAO-5766] Changes for inactive site redirection

**Please follow the below steps:**

1. Verify the latest changes(two files) and apply the same change in `CMSPencileblue` repo. Changes will be require manually on `CMSPencileblue`  repo for below files:
```
node_modules/@cbtn/pencilblue/include/http/middleware/routing.js
node_modules/@cbtn/pencilblue/include/http/middleware/index.js
```
2. Run the app.
3. Verify the changes.

**Testing Steps:**
1. Open https://premium.lvh.me:8080/admin/sites 
2. Deactivate Premium or Bravo.
3. Open below URLs to verify expected behaviour.
> https://premium.lvh.me:8080/bower_components/fontawesome/css/font-awesome.min.css
> https://premium.lvh.me:8080/bower_components/fontawesome/css/
> https://premium.lvh.me:8080/bower_components/fontawesome/
> https://premium.lvh.me:8080/bower_components/
4. All above URLs should redirect on https://www.careerbuilder.com/browse?TNNotFound=true
PFB the screenshot:
![image](https://user-images.githubusercontent.com/48584308/83230732-eada3200-a1a7-11ea-8361-546c785de8af.png)
